dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -61,6 +61,22 @@ def bulk_add
     end
   end
 
+  # Remove a student from the section
+  # POST /sections/:section_id/students/:id/remove
+  def remove
+    student = @section.students.find_by_id(params[:id])
```

Comment:
Now that this line is repeated now, here and in [update](https://github.com/code-dot-org/code-dot-org/blob/7138f9eced734c8c186d450e8d88895b9800f99e/dashboard/app/controllers/api/v1/sections_students_controller.rb#L18)). It's probably worth leveraging CanCan now.
Instead of doing this explicitly, add the following line to the top the class, just after the existing `load_and_authorize_resource`:

```ruby
load_resource :student, through: :section, only: [:update, :remove]
```

That will populate `@student` for both of those actions.